### PR TITLE
Reviewed A's

### PR DESCRIPTION
**Pull request by Murkantilism**
Reviewed Alabama, Arizona, and Arkansas.

Wanted to check what level of depth we are looking for @emarcey 

Also wanted to ask what to do about `az-phoenix-3`, where zip ties were almost certainly used but I can't find text confirmation on the net and all footage is too low quality or too dark to tell for sure.

### CODE REVIEW

File: reports/Alabama.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 Police arrested two AL.com journalists covering a protest at 7:35 on June 3. There was a 7PM curfew in effect, which exempted journalists who were carrying a letter from their media outlet; both arrested journalists were carrying such a letter and had press lanyards. Both journalists were released shortly thereafter; while in custody, they were photographed. Birmingham police later said that the arrest was made for the journalists' safety.
 
+Two employees of Alabama Media Group were restrained using zip ties.
+
 tags: arrest, journalist
 
-id: al-birmingham-1
+id: al-birmingham-1, zip-tie
```
Comment (emarcey) [inline reports/Alabama.md:11]: Wrong line

File: reports/Alabama.md [hunk 2]
```diff
@@ -4,9 +4,11 @@
 
 Police arrested two AL.com journalists covering a protest at 7:35 on June 3. There was a 7PM curfew in effect, which exempted journalists who were carrying a letter from their media outlet; both arrested journalists were carrying such a letter and had press lanyards. Both journalists were released shortly thereafter; while in custody, they were photographed. Birmingham police later said that the arrest was made for the journalists' safety.
 
+Two employees of Alabama Media Group were restrained using zip ties.
```
Comment (emarcey) [inline reports/Alabama.md:7]: Is this in the original video?

File: reports/Arizona.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ On May 31, police forced protesters who were out after curfew into the Garfield
 
 Several other residents of the Garfield neighborhood have complained of police behavior that night, including one who says she passed out in her yard due to a tear-gas-induced asthma attack and awoke to find an officer handcuffing her.
 
-tags: arrest, tackle, tear-gas, bystander
+tags: arrest, tackle, tear-gas, bystander, zip-tie-likely
```
Comment (emarcey) [inline reports/Arizona.md:52]: we don't do "likely." The goal here is clear evidence so if you're not 100% certain, don't tag it

**Review by emarcey — CHANGES_REQUESTED** [review]

File: reports/Alabama.md (hunk 2 above)
Comment (Murkantilism) [reply reports/Alabama.md:7]: There is no video of the arrest, the only video linked is a short clip of protestors attacking the reporters and vandalizing a storefront.

The article states:
> The two were restrained using zip ties. Koplowitz and Dunigan were placed in a transport van and taken to the city jail along with several other protestors who had been arrested. According to Koplowitz, the entire group was taken out of the van and handcuffed to a bench when they arrived at the city jail.

File: reports/Arizona.md (hunk 3 above)
Comment (Murkantilism) [reply reports/Arizona.md:52]: I know, it was a placeholder while I looked for more info & waited on your input.

I did finally find some high-resolution images in [an MSN article](https://www.msn.com/en-us/news/crime/phoenix-crowds-protest-deaths-of-george-floyd-dion-johnson/ss-BB14SxJ4?parent-title=more-than-200-arrested-by-phoenix-police-during-protests-on-sunday-night&parent-ns=ar&parent-content-id=BB14Taoq&fullscreen=true#interstitial=3) showing zip ties were used in both downtown Phoenix and the Garfield neighborhood.

Here are a couple of the clearer ones from that MSN album:

File: reports/Alabama.md (hunk 1 above)
Comment (Murkantilism) [reply reports/Alabama.md:11]: Fxied

**Review by emarcey — APPROVED** [review]

**Comment by Murkantilism** [comment]
Got access to the `bitchthot420` twitter account following and was able to review al-huntsville-2 and al-huntsville-3, no new tags needed so will merge this PR upstream and move on to the rest of alphabet.

**Pull request merged**